scanpy/tests/test_datasets.py

```diff
@@ -98,6 +98,12 @@ def test_visium_datasets(tmp_dataset_dir, tmpdir):
     mbrain_again = sc.datasets.visium_sge("V1_Adult_Mouse_Brain")
     assert_adata_equal(mbrain, mbrain_again)
 
+    # Test that downloading tissue image works
+    mbrain, mbrain_img = sc.datasets.visium_sge(
+        "V1_Adult_Mouse_Brain", return_image_path=True
+    )
+    assert mbrain_img == sc.settings.datasetdir / "V1_Adult_Mouse_Brain" / "image.tif"
```

Comment:
Could this also make sure the file exists, maybe even is an image?

Reply:
I'm now checking with `file` whether the file is a tiff